tests/web3/firefly_getCoverageData0.in.json

```diff
@@ -17,7 +17,7 @@
     "params": [
       {
         "address": "0xdf1db454e4dc226da2b0f8d471976180555355e2",
-        "code": "0x62FFFFFF60005260206000F3"
+        "code": "0x62FFFFFF60005260206000F3F3"
```

Comment:
I think if we were to add even another opcode here, the algorithm below wouldn't show it at all in the results.

Can we add that as a test (just append another `F3`, so that there are `F3F3F3`)?

Reply:
Why wouldn't it? The `"programs"` list gets created in the `initVM` rule (line 1425). Where the bytecode gets parsed and all the opcodes of the program are identified.